mediapipe/examples/desktop/face_mesh_dll/face_mesh_lib.cpp

```diff
@@ -408,6 +506,43 @@ node {
   output_stream: "DETECTIONS:face_detections"
   output_stream: "ROIS_FROM_DETECTIONS:face_rects_from_detections"
   output_stream: "FACE_COUNT_FROM_LANDMARKS:face_count"
+  output_stream: "IMAGE_SIZE:image_size"
+  output_side_packet: "ENVIRONMENT:environment"
+}
+
+# Applies smoothing to a face landmark list. The filter options were handpicked
+# to achieve better visual results.
+node {
+  calculator: "MultiFaceLandmarksSmoothingCalculator"
+  input_stream: "NORM_MULTI_FACE_LANDMARKS:multi_face_landmarks"
+  input_stream: "IMAGE_SIZE:image_size"
+  input_stream: "FPS:fps"
+  output_stream: "NORM_FILTERED_MULTI_FACE_LANDMARKS:filtered_multi_face_landmarks"
+  node_options: {
+    [type.googleapis.com/mediapipe.LandmarksSmoothingCalculatorOptions] {
+      velocity_filter: {
+        window_size: &window_size_param
```

Comment:
Why did you decide to use `&` here instead of `$` as in the previous cases?

Reply:
Ups, my fault)